content/spin/v2/quickstart.md

```diff
@@ -268,10 +267,8 @@ This command created a directory with the necessary files needed to build and ru
 $ cd hello_rust
 $ tree
 .
-├── .cargo
-│   └── config.toml
-├── .gitignore
 ├── Cargo.toml
+├── .gitignore
```

Comment:
nit: I'm not seeing this entry after I `spin templates upgrade` and `spin new -t http-rust`...

Reply:
It is created @vdice but looks like `tree` hides it by default